<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<BASE HREF="http://www.bioinformatics.vg/Methods/PCR-tutorial.shtml"><table border=1 width=100%><tr><td><table border=1 bgcolor=#ffffff cellpadding=10 cellspacing=0 width=100% color=#ffffff><tr><td><font face=arial,sans-serif color=black size=-1>This is <b><font color=#0039b6>G</font> <font color=#c41200>o</font> <font color=#f3c518>o</font> <font color=#0039b6>g</font> <font color=#30a72f>l</font> <font color=#c41200>e</font></b>'s <a href="http://www.google.com/intl/en/help/features.html#cached"><font color=blue>cache</font></a> of <a href="http://www.bioinformatics.vg/Methods/PCR-tutorial.shtml"><font color=blue>http://www.bioinformatics.vg/Methods/PCR-tutorial.shtml</font></a> as retrieved on Jan 7, 2008 20:39:14 GMT.<br>
<b><font color=#0039b6>G</font> <font color=#c41200>o</font> <font color=#f3c518>o</font> <font color=#0039b6>g</font> <font color=#30a72f>l</font> <font color=#c41200>e</font></b>'s cache is the snapshot that we took of the page as we crawled the web.<br>
The page may have changed since that time.  Click here for the <a href="http://www.bioinformatics.vg/Methods/PCR-tutorial.shtml"><font color=blue>current page</font></a> without highlighting.<br>
This cached page may reference images which are no longer available. Click here for the <A HREF="http://209.85.165.104/search?q=cache:hcT9KneLdQgJ:www.bioinformatics.vg/Methods/PCR-tutorial.shtml+plasmid+tutorial&hl=en&client=opera&gl=us&strip=1"><font color=blue>cached text</font></a> only.<br>To link to or bookmark this page, use the following url: <code>http://www.google.com/search?q=cache:hcT9KneLdQgJ:www.bioinformatics.vg/Methods/PCR-tutorial.shtml+plasmid+tutorial&amp;hl=en&amp;ct=clnk&amp;cd=38&amp;gl=us&amp;client=opera</code></font><br><br><center><font size=-2><i>Google is neither affiliated with the authors of this page nor responsible for its content.</i></font></center></td></tr>
<tr><td>
<table border=0 cellpadding=0 cellspacing=0><tr><td><font face=arial,sans-serif color=black size=-1>These search terms have been highlighted:&nbsp;</font></td><td bgcolor=#ffff66><b><font face=arial,sans-serif color=black size=-1>plasmid&nbsp;</font></b></td><td bgcolor=#a0ffff><b><font face=arial,sans-serif color=black size=-1>tutorial&nbsp;</font></b></td></tr></table>
</td></tr></table></td></tr></table>
<hr>
<div style="position:relative">
<?xml version="1.0" encoding="iso-8859-1"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<head><title>PCR tutorial for absolute beginners</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<meta content="PCR tutorial" name="description" />
<meta name="keywords" content="PCR, tutorial, PCR tutorial, beginners, thermocycler, primers, primer design"  />
<meta content="webmaster@molbiol.net" name="OWNER" />
<meta content="webmaster@molbiol.net" name="AUTHOR" />
<meta http-equiv="CONTENT-LANGUAGE" content="English" />
<meta http-equiv="VW96.OBJECT TYPE" content="Homepage" />
<!-- syntax for Pragma header: -->
<meta http-equiv="pragma" content="nocache" />
<!--+ for Expires header: -->
<meta http-equiv="Expires" content="Mon, 1 October 2000 00:00:01 PST" />
<!--or other time/date value,common to put date past to encourage cache 2 expire obj-->
<meta content="INDEX,FOLLOW" name="ROBOTS" />
<meta content="MSHTML 5.50.4134.100" name="GENERATOR" />
<link href="protocol_style.css" type="text/css" rel="stylesheet" />
</head>
<body bgproperties="fixed" bgcolor="#ffffff">
<!-- banner doesn't have to be an SSI//-->

<table border="0" cellpadding="0" cellspacing="0"><tr>

<td id="t0" bgcolor="#0383a0" width="60" nowrap>

<center><strong>
<A STYLE="text-decoration:none; font-size: 10pt; color:#ffffff; font-weight: bold;" TARGET="_top" HREF="http://www.bioinformatics.vg/index.shtml">
<FONT COLOR="#ffffff">&nbsp;Links&nbsp;</font></a></strong></center></td>

<td width="1" bgcolor="#808080"><img width="1" height="1" ALT="spacer" SRC="http://www.bioinformatics.vg/Images/transpix.gif"></td>

<td bgcolor="#efefef" width="60" nowrap><center><font size="-1" color="#0383a0"> 

<A STYLE="text-decoration:none" TARGET="_top" HREF="http://www.bioinformatics.vg/forum/"> &nbsp;Forum&nbsp; </a>
</font></center></td><td width="1" bgcolor="#808080">
<img width="1" height="1" ALT="spacer" SRC="http://www.bioinformatics.vg/Images/transpix.gif">
</td>

<td bgcolor="#efefef" width="60" nowrap><center><font size="-1" color="#0383a0"> 

<A STYLE="text-decoration:none" TARGET="_top" HREF="http://www.bioinformatics.vg/biolinks/biodirectory.shtml"> &nbsp;BioDirectory&nbsp; </a>
</font></center></td><td width="1" bgcolor="#808080">
<img width="1" height="1" ALT="spacer" SRC="http://www.bioinformatics.vg/Images/transpix.gif">
</td>

<td bgcolor="#efefef" width="60" nowrap><center><font size="-1" color="#0383a0"> 

<A STYLE="text-decoration:none" TARGET="_top" HREF="http://www.bioinformatics.vg/biolinks/search.shtml"> &nbsp;Search&nbsp; </a>
</font></center></td><td width="1" bgcolor="#808080">
<img width="1" height="1" ALT="spacer" SRC="http://www.bioinformatics.vg/Images/transpix.gif">
</td>

<td bgcolor="#efefef" width="60" nowrap><center><font size="-1" color="#0383a0"> 

<A STYLE="text-decoration:none" TARGET="_top" HREF="http://www.bioinformatics.vg/biolinks/news.shtml"> &nbsp;News&nbsp; </a>
</font></center></td><td width="1" bgcolor="#808080">
<img width="1" height="1" ALT="spacer" SRC="http://www.bioinformatics.vg/Images/transpix.gif">
</td>

<td bgcolor="#efefef" width="60" nowrap><center><font size="-1" color="#0383a0"> 

<A STYLE="text-decoration:none" TARGET="_top" HREF="http://www.bioinformatics.vg/biolinks/jobs.shtml"> &nbsp;Jobs&nbsp; </a>
</font></center></td><td width="1" bgcolor="#808080">
<img width="1" height="1" ALT="spacer" SRC="http://www.bioinformatics.vg/Images/transpix.gif">
</td>

<td bgcolor="#efefef" width="60" nowrap><center><font size="-1" color="#0383a0"> 

<A STYLE="text-decoration:none" TARGET="_top" HREF="http://www.bioinformatics.vg/cgi-bin/books/call_books.pl?keyword=Bioinformatics&amp;page=1&amp;used=new"> &nbsp;Books&nbsp; </a>
</font></center></td><td width="1" bgcolor="#808080">
<img width="1" height="1" ALT="spacer" SRC="http://www.bioinformatics.vg/Images/transpix.gif">
</td>

<TD></TD>
</tr></table>
<table width="100%" border="0" cellpadding="1" cellspacing="0" bgcolor="#3366cc">
<tr><td bgcolor="#0383a0" NOWRAP>
<table width="100%" border="0" cellpadding="1" cellspacing="0" bgcolor="#3366cc"><tr><td bgcolor="#0383a0" NOWRAP>
<h1 CLASS="header">Bioinformatics, Genomics, Proteomics, Biotechnology and Molecular Biology Directory</h1>
</td></tr></table>

</td></tr></table>

<!-- start of banner table-->
<TABLE WIDTH="100%" BORDER=0 CELLPADDING=0 CELLSPACING=0><TBODY><TR><TD COLSPAN=5 BGCOLOR=black>&nbsp;</TD></TR>
<TR>
<TD BGCOLOR=black>&nbsp;&nbsp;&nbsp;</TD>
<TD BGCOLOR=black ALIGN="LEFT" valign="top">
<IMG SRC="../biolinks/images/simple_20075.jpg" alt="MolBiol.Net&nbsp;-&nbsp;Probably&nbsp;the&nbsp;best&nbsp;bioinformatics&nbsp;and&nbsp;molecular&nbsp;biology&nbsp;portal&nbsp;on&nbsp;Earth">
</TD>
<TD BGCOLOR=black>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
<TD BGCOLOR=black>
<script type="text/javascript"><!--
google_ad_client = "pub-5891442971751245";
google_ad_width = 728;
google_ad_height = 90;
google_ad_format = "728x90_as";
google_color_border = "000000";
google_color_bg = "F0F0F0";
google_color_link = "0000FF";
google_color_url = "008000";
google_color_text = "000000";
//--></script>
<script type="text/javascript"
  src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
</script>

</TD>
<TD WIDTH="100%" BGCOLOR=black>&nbsp;</TD></TR>
<TR><TD COLSPAN=5 BGCOLOR=black>&nbsp;</TD></TR>
</TBODY></TABLE>
<!-- end of titlebar and banner table-->
<h1>PCR <b style="color:black;background-color:#a0ffff">Tutorial</b> for Absolute Beginners</h1>
<br>

<table width="95%" align="center" border="1" cellpadding="5" cellspacing="5" class="maintable">
  <tr align="center" valign="middle" >
    <td valign="middle">
	<a href="javascript:window.print()"><img align="absmiddle" src="../Images/printer.jpg" alt="Print this document" /></a>
    <strong> Click here to PRINT this protocol</strong>
	</td>
	<td valign="middle"><div align="right">If you wish to have one of your protocols listed here, 
        send it by email to <a href="mailto:protocols@molbiol.net"><font size="2">protocols@molbiol.net</font></a> 
      </div></td>
  </tr>
</table>

<p class="titlebar"> Reassuring words</p>
<p class="solutiontext">PCR is not alchemy, but when you start off it may seem 
  a little 'magical'.&nbsp; After a number of weeks or months, or sometimes to 
  start with, unexpected results can be obtained, or it can fail altogether.&nbsp; 
  In this short PCR <b style="color:black;background-color:#a0ffff">tutorial</b>, I will attempt to give a basics-only knowledge of 
  the reaction, and how to make it work.&nbsp; I will talk about the theory of 
  PCR, because you can read this in any textbook.&nbsp; I would refer you to the 
  seminal paper on PCR by Saiki R, K et al, "Enzymatic amplification of beta-globin 
  genomic sequences and restriction site analysis for diagnosis of sickle cell 
  anemia" <em>Science</em>, 1985 Dec 20, 230(4732):1350-4 which can be found <a target="_blank" href="http://sunsite.berkeley.edu/cgi-bin/ebind2html/pcr/034?seq=1">here 
  </a>and the abstract <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=2999980&dopt=Abstract" title="pubmed link for Saiki seminal PCR paper" target="_blank">here</a>.&nbsp; 
  This <b style="color:black;background-color:#a0ffff">tutorial</b> will be practical and aimed directly and the lab bench.</p>
<p class="solutiontext">There are seven components in a basic 
  PCR reaction, which should be added in the following order:</font></p>
<p class="solutiontext"><center>Buffer, nucleotides, water, first primer, 
  second primer, template and Taq.</center></p>
<p class="solutiontext">The buffer contains a salt, usually 
  KCl, as well as some magnesium chloride, which is a cofactor for Taq.&nbsp; 
  The nucleotides are deoxyribonucleotide triphosphates which are the building 
  blocks of DNA (A,T,C,G).&nbsp; Water should be of the highest grade, preferably 
  from a water purifier such as those made by millipore called '<a 
href="http://www.millipore.com/catalogue.nsf/docs/C7658">MilliQ</a>' - you can 
  always buy in PCR-grade water from Eppendorf (Cat 0032 006.302).&nbsp; The primers bind the target 
  and make the reaction specific to that target.&nbsp; The template material is 
  the DNA which the primers will bind which contains the target sequence.&nbsp; 
  The Taq is the enzyme which carries out the amplification reaction by extending 
  the primers from their binding-sites on the target along the template.&nbsp; 
  We will now consider aspects of each component in a PCR reaction.</p>
<p class="titlebar">PROTOCOL FOR a 50 microlitre reaction</p>
<p class="protocoltext"> The following amounts and concentration should be adjusted 
  after opimization, but provide a good starting point.
<ol>
  <li>Add 5 ul 10 x Taq buffer</li>
  <li>Add 1 ul 10mM dNTP mix</li>
  <li>Add 40.8 ul of water</li>
  <li>Add 1 ul of each primer (10 pmol / ul)</li>
  <li>Add 1 ul template - pipette well</li>
  <li>Add 0.2 ul of Taq (5 units / ul)</li>
  <li>Store on ice until transfer to pre-heated thermocycler</li>
</ol>
<p class="titlebar">SOLUTIONS</p>
<p class="titlebarprotocol">Buffer</p>
<p class="solutiontext">The buffer must be thawed completely before use, and then 
  mixed.&nbsp; The salts can form concentration gradients during freezing and 
  thawing.&nbsp; When the contents is liquid, vortex it hard for ten&nbsp;seconds, 
  and have a good look at the tube to make sure there is <em>no </em>ice left.&nbsp;&nbsp; 
  I like to thaw all components in a polystyrene float in some sterile water in 
  a beaker, it takes 5-10 minutes, rather than 20 minutes in air, or much longer 
  on ice.&nbsp; People who thaw reagents slowly on ice usually seem to be demotivated, 
  and spend the hours it takes to thaw things drinking coffee.&nbsp; If you pipette 
  thawed liquid from the top of the tube's contents while most of it is still 
  frozen, you are making a mistake - not only will it contain less magnesium chloride, 
  by taking away water, it makes the contents more concentrated for both salt 
  and magnesium chloride.&nbsp; I always get "10 times concentrated" buffer containing 
  15mM MgCl<sub>2</sub>, although you can get the two separate for 'fancier' reactions, 
  which is nearly never necessary.&nbsp; Increasing the magnesium concentration 
  reduces specificity, generally one might try a reaction at a final magnesium 
  concentration of 1.5, 1.8, 2.1, 2.3, 2.5 mM.</p>
<p class="titlebarprotocol">Nucleotides</p>
<p class="solutiontext">I always buy my nucleotides from Promega, which simplifies 
  things.&nbsp;&nbsp;Make or buy (Promega cat#&nbsp;C1141)&nbsp;a stock solution 
  of 'dNTPs' in which each of the 4 nucleotides is at a concentration of 10mM.&nbsp; 
  This means I need to put 1 microlitre of dNTP (10 mM) in each 50ul reaction.&nbsp; 
  Again, thaw this well, and vortex well before use.&nbsp; I usually buy a concentrated 
  stock and dilute it myself (Promega cat# U1330) because its slightly cheaper.</p>
<p class="titlebarprotocol">Water</p>
<p class="solutiontext">Although double-distilled water may be fine, always autoclave water.&nbsp; 
  Ultrapure water such as that produced by milliQ machines contains virtually 
  no salts which could affect the PCR reaction adversely.&nbsp; Autoclaving water 
  does not get rid of contaminating DNA templates, so if the water is from a suspect 
  source, autoclaving does not solve anything.&nbsp; Serious problems with contamination 
  of water by <em>E.coli</em> can happen in nearly all labs, if the bugs in question 
  contain a <b style="color:black;background-color:#ffff66">plasmid</b> in which you had previously cloned a target sequence, you 
  will be in trouble.&nbsp; Keep a set of glass bottles (eg. Schott /Duran) specifically 
  for the purposes of ultrapure water storage.&nbsp; Never dip a pipette into 
  these bottles, always pour your 'working "solution" of ultrawater' into a microfuge 
  tube.</p>
<p class="titlebarprotocol">Primers</p>
<p class="solutiontext">I order my primers from <a target="" href="http://www.genosys.com">Genosys</a> 
  because they're cheap, fast and good quality.&nbsp; I order 5 micromole scale.&nbsp; 
  The primers are supplied lyophilized (dried) as an off-white pellet in the base 
  of the tube.&nbsp; On the side of the tube, it will tell you the amount of primer 
  in the tube in nanomoles (eg. 78 nmol).&nbsp; I always resuspend the primer 
  in the same number of microlitres of TE (eg. 78 microlitres is used to resuspend 
  78nmoles) - 10 mM Tris-HCl, 1 mM Na<sub>2</sub>EDTA, pH 8.0.&nbsp; This means 
  your stock is at a concentration of 1 mM.&nbsp; Vortex for about 1 minute, then 
  spin down briefly in a microfuge.&nbsp; I then go on to make a 'working solution' 
  in water, at a final concentration of 10 mM, which is the same as 10 pmol / 
  ul.&nbsp;</p>
<p class="titlebarprotocol">Template concentration 
  and quality</p>
<p class="solutiontext">As a rule of thumb you want about 50,000 
  copies of whatever you are amplifying in your reaction to begin with.&nbsp; 
  If you are using a crude template, such as a colony, 1 microlitre of a suspension 
  of a colony in 30 microlitres of water is sufficient. The final concentration 
  of template should be less than 10ng per microlitre.&nbsp; Otherwise the templates 
  amounts listed below are a starting point.</p>
<dl>
	<dd><b style="color:black;background-color:#ffff66">Plasmid</b> 50fg</dd>
	<dd>Prokaryotic genomic DNA&nbsp; 0.1ng</dd>
	<dd>Eukaryotic genomic DNA&nbsp;&nbsp; 50-100ng</dd>
</dl>

<p class="titlebarprotocol">Taq DNA polymerase</p>
<p class="solutiontext">I use Sigma 
  Taq, which is cheap, but I have used Gibco, Pharmacia, but I particularly liked 
  Promega Taq with Buffer B which contains a detergent, probably Triton X100.&nbsp; 
  Get it from a reputable supplier, and use no more than 1 unit per 50 microlitre 
  reaction, scaling down with reaction volume as with all the other components.&nbsp; 
  Add this last!</p>
<p class="titlebar">Thermocycles</p>
<p>Although it is not possible to predict what PCR cycling conditions will produce best results first time, the following is a rule of thumb which might be useful as a starting point.
The annealing temperature can be calculated using <a href="http://www.bioinformatics.vg/cgi-bin/biolinks/redirect.cgi?http://www.bioinformatics.vg/bioinformatics_tools/oligo2002.shtml&id=236" target="_blank">Oligo 2002</a>.  
<dd>
	<dl>Initial cycle: 1 minute at 95&deg;C</dl>
	<dl>40 cycles of:</dl>
	
  <dl>
    <dl>
      Denaturation: 1 minute at 95&deg;C 
    </dl>
		<dl>Annealing:   1 minute at 50&deg;C</dl>
		<dl>Extension:   1 minute at 72&deg;C</dl>
		<dl>Final extension: 10 minutes at &deg;C</dl>
</dl>
</dd>
The extension cycle should be extended by 1 minute per kilobase of DNA to be amplified, ie. a two minute extension is required for a 2kb target.
 
</p>
<p class="titlebar">Primer design</p>
<p>Primer design is an art when done by human beings, and a far better done by 
  machines.&nbsp; Use one of the programs available on the <a href="http://www.bioinformatics.vg/biolinks/bioinformatics/verbose/PCR%2520and%2520Primer%2520Design.shtml">portal</a> 
  to design your primers. You can partly insure against unexpected aberrant PCR products by searching 
  the public gene databases for the primer nucleotide sequences using <a 
href="http://www.ebi.ac.uk/fasta33/">FASTA</a>&nbsp;for example.&nbsp; If there 
  similar or identical sequences in the database, they may provide an alternative 
  target during the reaction itself.&nbsp; There are exquisite tools for the analysis 
  of primers, which human beings can use, but take the output from machines generally 
  takes primer dimer and primer self-dimers into account. A list of these 
  is available <a href="http://www.bioinformatics.vg/biolinks/bioinformatics/verbose/PCR%2520and%2520Primer%2520Design.shtml" title="pcr links" target="_blank">here</a></p>
<p class="titlebar">Equipment Considerations</p>
<p align="left">Tubes - size, thin walled.&nbsp; PCR blocks 
  usually take either upto&nbsp;forty 0.5mL tubes, ninety six 0.2mL tubes...there 
  are blocks that take 1mL tubes and others that take 384 well plates.&nbsp; For 
  most applications 0.2mL tubes are ideal, since most reactions will be less than 
  or equal to 50 microlitres in volume.&nbsp; The downside is the smaller the 
  tube, the lower the centrifugal force before it falls apart.&nbsp; Techne sell 
  relatively robust 0.5mL tubes which survive centrifugation at 10,000rpm in a 
  bench-top 'fuge.&nbsp; I prefer thin walled 0.2mL tubes because they seem to 
  give good results on short thermocycling programs.</p>
<p align="left">Use a heated lid cycler - don't bother with 
  mineral oil, it can inhibit your reactions.&nbsp; If you do use it, keep it 
  in a cupboard - there is a lab 'myth'&nbsp; that UV light causes the formation 
  of free-radicals in the mineral oil, which will completely inhibit your reactions.&nbsp; 
  Besides, its messy, impossible to pipette reprocibly, absorbs too much heat, 
  and a certain source of cross contamination.</p>
<p align="left">Thermocycles - most applications require 30-35 
  cycles, depending on efficiency and template abundance.&nbsp; Amplifying a cDNA 
  might require 35 cycles, whereas checking for an insert in a&nbsp;<b style="color:black;background-color:#ffff66">plasmid</b> might 
  require 20 or fewer cycles.&nbsp; I generally start with 94C for 45 seconds, 
  annealing temperature for 1 minutes, 72C for 1 minute per 500bp of target PCR 
  product.&nbsp; The annealing temperature should be 5C below the melting temperature 
  of the lowest primer.</p>
<p align="left">Thermocycler&nbsp;&nbsp; If you have money, 
  use a DNA Engine (MJ Research) - I use a PTC-200 gradient block (about Â£5000), 
  otherwise try to get a few on trial for 2 weeks to 'road test' them from a number 
  of suppliers before you spend your grant money!&nbsp; Remember - fast peltier 
  blocks are best (3 degrees Celsius per second heating)! Gradient blocks allow 
  you to use a range of annealing temperatures across the block, allowing fast 
  determination of optima as well as allowing reactions requiring different annealing 
  temperatures to be run at the same time.</p>
<p align="left">Pipetting - Use a 10 microlitre pipette for small volumes, with 
  10 or 20 microlitre tips.&nbsp; 'Tip off' excess Taq - adding more will only 
  reduce specificity.</p>
<p class="titlebar">Other Considerations</p>
<p>Everying is stored at -20 degrees Celsius 
  or colder, or on ice when thawed for use.&nbsp; Buy your reagents from reputable 
  companies like Promega,&nbsp;Life Technologies (Gibco), and Genosys.<br />
</p>

<BR><BR><BR><SCRIPT SRC="http://www.bioinformatics.vg/js_scripts/antispam.js" TYPE="text/javascript"></SCRIPT>
<center>
  <a href="http://www.bioinformatics.vg/index.shtml">Home</a> | <a 
href="http://www.bioinformatics.vg/biolinks/forum.shtml">Community</a> | <a 
href="http://www.bioinformatics.vg/recommend_to_a_friend.shtml">Recommend Us</a> 
  | <a href="http://www.bioinformatics.vg/help.shtml">Help</a> | <a 
href="http://www.bioinformatics.vg/about.shtml">About</a> <br />
  <a 
href="http://www.bioinformatics.vg/contact.shtml">Contact Us</a> | <a 
href="http://www.bioinformatics.vg/advertise.shtml">Advertise</a> | <a 
href="http://www.bioinformatics.vg/addasite.shtml">Add a Site</a> | <a 
href="http://www.bioinformatics.vg/disclaimer.shtml">Terms and Conditions</a> 
  | <a href="http://www.bioinformatics.vg/w3c/index.shtml">Privacy Statement</a> 
  | <a href="http://www.bioinformatics.vg/bookmark.shtml">How to Bookmark</a> 
  <br />
  Questions? <SCRIPT language="JavaScript" TYPE="text/javascript">
<!--
write_email('webmaster','bioinformatics&#40;hotmail&#41;net&#40;com&#41;','Contact&#32;Us');
//-->
</SCRIPT><br />
                Copyright &copy; 2003-2005 Oxford Informatics Ltd and Bioinformatics.Net - Sponsored by Invitrogen<br /> </center><BR><BR><BR>

<p align="right"><img src="../biolinks/images/molbiol100x30.jpg" /></p>

</body></html>
