<HTML> <TITLE>Tips For Oligonucleotide Synthesis</TITLE> <P align =left><p>

<h1>Tips For Oligonucleotide Synthesis</h1><br>
<h3>Richard T. Pon -
University of Calgary<br></h3><p>

<hr>





Oligonucleotide synthesis is by far the most common nucleic acid service
provided by core facilities. The chemistry and instrumentation required has
been extensively optimized, and excellent results can be obtained by almost
anyone with a DNA synthesizer and good quality reagents. Indeed, the
synthesis of short (less than 40 nucleotide) probes and primers has become
so easy that no special expertise, other than the ability to read the
operator's manual is required, and a part-time operator with a single or
double column DNA synthesizer can handle the requirements of a small group.<p>

However, core service facilities must be able to make thousands of
sequences and do it quickly, reliably, and economically, and this requires
somewhat more skill and expertise. In addition some core facilities, such
as mine, may be asked to prepare sequences that are more difficult than the
usual probes and primers. I believe that there are two ways good core
service facilities can distinguish themselves from smaller synthesis
operations, such as the individual lab synthesizer. First, core services
should be able to make a great many sequences of consistently high quality
and without mistakes. Secondly, the service should be able to handle
"difficult'' requests, such as long sequences or sequence modifications
that may be beyond the scope of smaller operations.<p>

In this article I would like to present some suggestions that can help meet
these criteria as well as assist those who may be encountering difficulty
with their syntheses. Please keep in mind that these are only suggestions
and not every facility will find them necessary or desirable. However, they
have all proven to be useful in my laboratory, which specializes in
preparing difficult oligonucleotides and developing new synthetic
methodology.<p>

<ol><li> <b>Check Phosphoramidites for Coupling Efficiency</b><p>
Phosphoramidites are the most important reagents, and good results cannot
be obtained with batches that have not been carefully purified or stored
free of moisture. Over the last nine years, our laboratory has checked
phosphoramidite lots for coupling efficiency before releasing them for
general use. This is because we have observed significant differences in
the coupling ability between different batches and have occasionally found
lots that have had to be rejected outright because of either poor coupling
performance (less than 98%) or solubility problems. Lot-to-lot variability
has improved over the last few years, but core facilities should be aware
that different lots are not always equal.<p>

In our laboratory we test each phosphoramidite lot by preparing a 12 base
long sequence containing only that base. The trityl colors from each
coupling step are collected and carefully quantitated using either a
colorimeter or spectrophotometer (not an on-line trityl monitor), and the
average coupling yield is calculated. Because this assay deals with
relatively small changes in coupling efficiency, it is very important that
the trityl measurements by accurately performed, that other reagents be
fresh, and the instrumentation is working correctly. This allows us to
determine the average coupling efficiency of a particular lot under our
conditions and on our own synthesizer. Most manufacturers test for coupling
ability, but these methods vary and I think that the only test that matters
is the one done on your own instrument. It is also possible that traces of
moisture introduced in the packaging step or improper handling during
shipment can affect the final coupling efficiency.<pi>

The average coupling efficiency of a good lot should be greater than 98.5%,
with very good preparations producing average yields of greater than 99%.
Lots producing coupling yields less than 98.0% are not accepted. Because
higher coupling efficiencies produce a higher overall yield, using a good
or better than average lot can significantly improve oligonucleotide
quality. This difference may not be noticeable with short oligonucleotides,
but the effect becomes more pronounced as chain length increases, and the
higher overall yield can determine whether a sequence will work without
purification or even whether a long sequence can be isolated.<pi>

In the past, I have requested that manufacturers send me samples of
specific lots for preapproval before purchase. In this manner, I have been
able to select only the best phosphoramidite lots, and I would usually
purchase a year's supply at one time (There is no loss of activity when
storing phosphoramidites in the freezer for this period of time). However,
since production methods seemed to have improved in the last few years, I
no longer feel that testing before purchase is necessary. We do, however,
continue to check each new lot as it arrives. While this may not be
necessary for those facilities that make only short sequences, I recommend
this testing for those facilities having difficulty with long
oligonucleotides or for those who want to consistently produce the best
possible material.<p>

<li> <b>Monitor Synthesizer Performance with Trityl Analysis</b><p>
The orange-colored trityl cation produced by the dimethoxytrityl protecting
group in acid provides a convenient quantitative method for determining
coupling efficiency. This is because of the one-to-one relationship between
the number of trityl molecules and the number of new phosphate linkages
formed. Unfortunately, the quantitative nature of this measurement has been
dismissed by some as only "approximate". This could be due to two factors.
First, early CPG supports contained improperly blocked surface functions
that could lead to formation of new oligonucleotide chains and apparent
trityl "yields" of well over 100% (1). However, present CPG and polystyrene
supports (2) no longer have this problem. Second, some instruments do not
reproducibly deliver all the trityl cation into the fraction collector and
so the trityl measurements are not accurate. In my past work, developing
both synthesis instrumentation and reagents, I have found that careful
collection and measurement of trityl colors is a reliable assay of coupling
efficiency.<p>

Trityl analysis, on a daily basis, is important for a core synthesis
facility because it provides an early indication of how well the coupling
reactions are proceeding. Although the trityl results cannot detect
problems with poor oxidation or capping steps, they do warn of problems
with moisture contamination, poor coupling reagents, blocked or partially
plugged lines, and a variety of other instrument malfunctions. In addition,
if the correct sequence has been entered into the synthesizer and the
correct number of good trityl colors has been verified, then one can be
assured that the correct sequence has been made and only a gel picture or
chromatogram confirming sequence homogeneity (i.e., correct capping and
oxidation) is required for complete quality control documentation.<p>

Simple visual inspection of the trityl colors can alert an operator to an
immediate and outright malfunction, but only quantitative measurements can
detect smaller reductions in instrument performance. In a busy facility, it
is important to catch these failures as soon as possible in order to avoid
delays or the inadvertent production of a large number of problem
oligonucleotides. Unfortunately, however, accurate trityl analysis on a
spectrophotometer is much too labor intensive for routine use by any core
facility, and therefore a variety of other methods for trityl analysis,
which are easier although less accurate, have been developed. I would like
to describe how our laboratory uses three of these alternative methods for
trityl analysis in our routine synthesis operations. These are described in
order of increasing complexity and accuracy.<p>

The first method, we have used, is the AutoAnalysis option available on
Applied Biosystems 392/394 DNA synthesizers (3). This option consists of
individual conductivity detectors (one per column) that measure the amount
of charged trityl cation flowing out of each synthesis column. The ABI
monitor is easy to use because it is integrated into the synthesizer.
However, it is expensive and does not report the coupling yields for each
individual step (although it measures and calculates them). Instead, an
average stepwise yield, which masks individual step yields into one average
value, is presented for each step. This makes it difficult to decipher the
actual machine performance and so the detector is limited to the detection
of rather severe failures.<p>

A more accurate detector, the TritylTech on-line trityl detector, is
available from Ana-Gen Technologies (4) and can be attached to almost any
DNA synthesizer, including the ABI 380A/B series. This detector employs the
more conventional colorimetric method for measuring trityl colors but
differs from the flow-through trityl detectors used by Pharmacia, Milligen
and other vendors (I have not evaluated any of the non-ABI synthesizers) by
using up to four quartz cuvettes to collect the trityl colors. The color
measurements only takes place after the entire colors have been collected,
partially diluted, and mixed. Coupling data is presented in a clear, easily
understood format (percent stepwise and overall yields), and the monitor is
capable of detecting couplings that are only a few percentage points lower
than normal. However, the detector is not integrated with the synthesizer
and requires a separate PC controller that must be configured prior to each
synthesis. Obtaining a printout of the results is difficult because the
monitor uses a parallel port connection for data acquisition instead of a
serial port, and the time required to flush out the quartz cuvettes adds
almost a minute to each coupling cycle.<p>

The third and oldest method of trityl analysis employed in our laboratory
is the most time consuming but offers accuracy similar to a
spectrophotometer. This method utilizes a PC-800 fiber-optic "dipping
probe" colorimeter from Brinkmann Instruments (5), which eliminates the
need to transfer cuvettes in and out of a spectrophotometer. This makes the
measurements much faster although still not automated. We use this
technique to measure the results from our older 380B and 38lA DNA
synthesizers by: a) collecting the trityl colors in 12 ml tubes, b)
diluting the tubes to a constant volume (about 7-8 ml) with 5%
dichloroacetic acid/1,2-dichloroethane using a bottle top dispenser, c)
vortexing, and d) dipping the fiber optic probe into each tube to measure
the absorbance. Either a 470 nm or 545 nm filter is used in the colorimeter
so that the absorbance reading will be within range. We usually measure
only the first three and last three trityl colors, unless we are testing
new reagents, synthetic procedures, or troubleshooting a problem, in which
case we check each coupling step. This method is probably the least
expensive and most effective method for those facilities who may only want
to perform the occasional trityl analysis.<p>

Finally, there has also been a semi-automated method (6) described that
uses a 96-well plate reader, which might be of interest to those
laboratories with an existing plate reader. However, this method has not
been evaluated in our laboratory.<p>

The present generation of automatic trityl detectors are not perfect, but
they are still an important way to monitor coupling efficiency, and I hope
that instrument manufacturers will try to improve the accuracy and ease
with which these methods work. In the meantime, the automated detectors are
still better than nothing, and large core facilities should consider them
as an easy method for the daily monitoring of their synthesizer
performance.<p>

<li><b>Use Dichloroacetic Acid for Detritylation</b><p>
Depurination (cleavage of the glycoside bond) under acidic conditions is an
important side reaction that limits how oligonucleotides can be synthesized
and so a great deal of work has gone into finding the optimum conditions
for detritylation. Originally, strong protic acids such as benzenesulphonic
acid (pKa = 0.5) or trichloroacetic acid (TCA, pKa = 0.7) were used.
However, in 1983, two groups independently found that when controlled pore
glass (CPG) was used as the solid-phase support a weaker acid,
dichloroacetic acid (DCA, pKa = 1.48), could be satisfactorily employed (7,
8). Indeed, when combined with phosphoramidite chemistry this combination
allowed the synthesis of a 5l base long oligonucleotide, a length much
greater than anything previously attempted at that time. DCA is also more
stable towards decomposition to hydrochloric acid than TCA (9). Since then,
DCA has been considered as the best reagent for detritylation, and many
publications using this acid at concentrations of either 1% (10), 2% (11),
2.4% (12), 3% (13), or 5% (14) (v/v) in either dichloromethane (DCM) or
1,2-dichloroethane (DCE) have appeared in the literature. This acid is also
used commercially on Milligen/Biosearch, Pharmacia (1.3 mmol scale), and
Applied Biosystems 390Z (large scale) DNA synthesizers (15). Although TCA
can cause substantial depurination in silica bound oligonucleotides (3%
TCA/DCM can cause 12-67% depurination in 1 hour, depending on the position
of the deoxyadenosine) (16), this reagent has continued to be sold for use
on Applied Biosystems and Beckman synthesizers.<p>

Obviously, the many millions of satisfactory oligonucleotides that have
been made with TCA are proof that this reagent is adequate for most
syntheses. However, when our laboratory began making longer
oligonucleotides (80-150 bases) as well as larger scale syntheses (10-15
mmol), we found that much better results were obtained with DCA instead of
TCA (data not shown), and we now use DCA solutions for all our syntheses.
In addition to reduced depurination, DCA solutions are also more
conveniently prepared than TCA solutions because DCA is a liquid instead of
a solid (we simply add the liquid acid directly to a 41 bottle of solvent
and filter). In our laboratory we use 5% (v/v) DCA/DCE for any sequence up
to approximately 40 bases long and only 2% (v/v) DCA/DCE for longer
sequences. It is not necessary to increase the total time for the
detritylation step (which is usually about 50-60 sec) because of the weaker
acid, but we find it is necessary to compensate for the lower flow rate of
the DCE solvent (DCE is more viscous than DCM). This can be easily done by
extending the delivery time of the reagent to the column so that enough
reagent reaches the synthesis column. Usually the total detritylation time
can be kept constant by decreasing any wait steps in the detritylation. We
prefer to use DCE as the solvent because it produces less depurination, it
is less toxic, and it lasts almost twice as long on the synthesizer because
of the lower volume consumed per cycle. The DCA/DCE combination also seems
compatible with the Applied Biosystems AutoAnalysis conductivity detector
(DCE must also be used on position 19), but we have not attempted to verify
the accuracy of the results obtained using this reagent combination.<p>

<li><b> Use Low Loading CPG for Long Oligonucleotides</b><p>
The synthesis of long (50-150 bases) oligonucleotides can be greatly
improved by using a CPG support (17) with a much lower loading (about 5
mmol/g) than the usual 30-40 mmol/g loading. The pore size of the support
must also be l,000 A or greater. CPG supports with pore sizes greater than
1,000 A are available, but we have always had good results (in the 50-150
base long range) with 1,000 A supports. We usually hand pack about 20 mg
(0.1 mmol (about 5 mmol/g)) of these supports into a synthesis column, but
twice as much can be squeezed into a standard synthesis column if
necessary. Synthesis is performed using our standard 0.2 mmol scale
synthesis cycle with 2% dichloroacetic acid/dichloroethane as the
detritylation reagent and no other modifications. We believe the lower
surface loading of the support gives much more consistent coupling yields,
perhaps because of less surface crowding or just the greater excess of
reagent present.<p>

These long oligonucleotides are purified using a 40 cm Bio-Rad Sequi-Gen
sequencing gel apparatus, which has been modified with 1.5 mm thick spacers
and combs (custom made in our machine shop). This apparatus allows 12%
polyacrylamide/7M urea denaturing gels to be run at 55°. The full length
product, which is the strongest band on the gel, is easily identified by UV
shadowing, and it can be cut out, extracted, and desalted in the usual way.<p>

<li><b> Make Long Degenerate Sequences With Premixed Bases</b><p>
Recent advances in combinatorial or "irrational" drug design require large
pools of random sequences from which specific sequences can be retrieved
(18). Typically these pools consist of very long oligonucleotides
containing many degenerate sites (i.e., 50-100 consecutive degenerate
positions). Most synthesizers can be programmed to mix degenerate bases
on-line, but we have found that sequences containing long stretches of
degenerate sites can be significantly improved by manually mixing equal
volumes of the four different phosphoramidite solutions together and
placing this mixture on a spare base position. A mixed base reagent can
also be made by weighing out portions of the solid phosphoramidites, but
the different molecular weights of each base (dA, 858; dG, 840; dC, 833; T,
744) must be considered to get equimolar amounts.<p>

The premixing ensures that the degenerate products created have a more
uniform base distribution than that created by automatic on-line base
mixing (19). When large numbers of degenerate bases are mixed
automatically, the amount of full length product that can be identified on
a gel is greatly reduced, often to the point that no product can be found
by UV shadowing. However, premixing the bases yields products of similar
quality to the nondegenerate sequences we make (data not shown). This is
because the coupling efficiency is slightly less when five bottles deliver
reagents to the column (four bases plus tetrazole) than when only two
bottles are used. Apparently, the decrease in coupling is so small that
only sequences with a very large number of degeneracies are affected, so
automatic mixing still produces good results for short, degenerate
oligonucleotides. However, when long random sequences are required, manual
mixing will allow the oligonucleotides to be produced in higher overall
yields, and the number of sequences in the random pool will be increased.<p>

<li><b> Use Good Quality Anhydrous Acetonitrile to Dissolve Phosphoramidites and
Tetrazole</b><p>
Laboratories in humid locations or those who manually dissolve their
phosphoramidite or tetrazole reagents should consider setting up their own
acetonitrile still so that a source of dry solvent is always available.
Commercially available anhydrous acetonitrile is good when originally
opened, but repeated sampling or prolonged storage of opened bottles can
introduce moisture contamination. Trace amounts of moisture lower the
effective phosphoramidite concentration (20) and hence coupling efficiency.
Therefore, the quality of anhydrous acetonitrile used for dissolving
phosphoramidites is very important.<p>

A simple solvent repurification apparatus consisting of a three neck flask,
a one piece distillation head/reservoir (1,000 ml) and a condenser can be
obtained commercially or from any glassblowing shop. We use a relatively
large still because we also prepare our own anhydrous tetrazole solutions.
Laboratories only requiring solvent for dissolving amidites can use a much
smaller apparatus. It is best to only use leftover anhydrous acetonitrile
in the still because this material already has a very low water content.
Removal of large amounts of moisture from acetonitrile requires a two stage
distillation process, once from phosphorus pentoxide and once from calcium
hydride and is much more cumbersome. However, a simple continuous reflux
over calcium hydride and under nitrogen is sufficient to keep the solvent
anhydrous. A single three-way stopcock switches the still head from reflux
into collection mode, and anhydrous solvent can be easily dispensed. The
entire apparatus takes up little space and requires virtually no
maintenance. We top up the reservoir with leftover acetonitrile every
couple of weeks, and once or twice a year we empty the apparatus to remove
excess calcium hydride and any impurities that may have slowly built-up.
This apparatus is less expensive than purchasing small bottles of anhydrous
acetonitrile, and fresh anhydrous solvent is available at all times.<p>

<li><b>Use All Plastic Syringes to Dispense, Transfer, and Filter Solutions</b><p>
Disposable syringes are very convenient for transferring small volumes of
reagents. However, most syringes contain a black rubber tip on the end of
the plunger that is not resistant to organic solvents, and contact with
acetonitrile or THF will "freeze" the plunger in place. This problem can be
avoided by using all plastic syringes (available from Aldrich or Sigma in
sizes from 1 to 50 ml) that are made only of polypropylene and
polyethylene. These are ideal for dissolving phosphoramidites and for
transferring phosphoramidite solutions from one bottle to another (remember
to wear safety glasses). Although the syringes are sold as sterile,
individually packaged items, they are inexpensive enough to be considered
as disposable items and we do not attempt to reuse them. If they are
treated as single-use items, we believe no special drying precautions are
required because the sterile syringes are clean and free of moisture. An
empty syringe filled with a drying agent ("Drierite") makes a convenient
vent for solution transfers and prevents exposure to atmospheric moisture.
Self-sealing white silicone rubber septa (available from Aldrich) also make
convenient caps for any leftover reagents because no aluminum seals are
required. Small amounts of reagent can also be easily filtered without
exposure to moisture by using these syringes and readily available syringe
filters as long as the filter units are compatible with solvents (i.e.,
avoid units with polystyrene or cellulose components).<p>

<li><b> PCR Can Be Used to Isolate Very Long Oligonucleotides</b><p>
Very long oligonucleotides can sometimes be synthesized if PCR is used to
amplify the full-length product from the complex reaction mixture that is
produced. In our laboratory we have successfully obtained sequences 250
bases long in this manner, and there are examples in the literature of
sequences in the range of 300-600 bases being obtained (21).<p>

<li><b>Verify the Presence of 5'-Amino or 5'-Biotin Groups</b><p>
Oligonucleotides containing 5'-amino groups or 5'-biotin groups are
required for many applications involving non-isotopic detection. The
successful incorporation of these end groups is essential for their
function, but there is no easy way to verify the presence of these groups
by either trityl analysis (aminolink reagents lack a trityl group, and
biotin reagents have a much slower rate of detritylation), electrophoresis,
or chromatography. This difficulty is compounded by the fact that the
end-modifying reagents are usually not frequently used and are often of
lesser quality (in terms of coupling efficiency) than the more common
phosphoramidites. The reagents are also much more expensive.<p>

Fortunately, the presence of these groups can be quickly determined by
using either ninhydrin or 4-dimethylamino-cinnamaldehyde (22) spray
reagents. This test is performed by using a small glass capillary, drawn to
a fine tip in a Bunsen burner, to spot about l ml of sample (about 0.l-0.5
ODU) onto a TLC plate. The TLC plate is only used because it is a
convenient substrate (we use the same fluorescent silica coated plastic TLC
sheets that we use for UV shadowing), and other substrates might also be
suitable. The sample is then sprayed with either ninhydrin (0.2% in
ethanol) or 4-dimethylaminocinnamaldehyde (1:1 mixture of 2% H2SO4 in
ethanol and 0.2% pDACA/ethanol) solution. The plate is then heated using a
hair dryer to develop the color. Ninhydrin produces dark blue spots if
amino groups are present, and the 4-dimethylaminocinnamaldehyde forms
pinkish-purple spots when biotin is present.<p>

Although these are only qualitative tests, they can determine whether or
not a 5'-modification is present. This is particularly important when
post-synthesis derivitization reactions fail, and it is necessary to
determine whether the oligonucleotide or the derivitization reagent (typical
ly an NHS ester) is at fault. They are also convenient for verifying that
the correct product has been isolated, whenever synthesis produces more
than one major product (e.g., from poor couplings or secondary structure).<p>
</ol>

References
<ol>
<li>	R. T. Pon, N. Usman, and K.K. Ogilvie, 1988, Derivatization of
controlled pore glass beads for solid phase oligonucleo-tide synthesis,
Biotechniques 6, 768-775.
<li>	C. McCollum and A. Andrus, 1991, An optimized polystyrene support for
rapid, efficient oligonucleotide synthesis, Tetrahedron Letters 32,
4069-4072.
<li>	J. Kaufman, M. Le, G. Ross, P. Hing, M. Budiansky, E. Yu, E. Campbell,
V. Yoshimura, V. Fitzpatrick, K. Nadimi, and A. Andrus, 1993, Trityl
monitoring of automated DNA synthesizer operation by conductivity: A new
method of real-time analysis, Biotechniques 14, 834-839.
<li>	Ana-Gen Technologies Inc., 1134 Aster Ave., Suite K, Sunnyvale, CA
94086. Tel: (408) 249-4362, Fax: (408) 249-4383.
<li>	In Canada, contact Brinkmann Instrument Ltd, 6670 Campobello Road,
Mississauga, Ont, L5N 2L8. Tel: (416) 826-5525, Fax: (416) 826-5424.
<li>	W. Lint, E. Vanaja, F. J. Grant, P.G. Lockhart and P.J. O'Hara, 1994,
Easier DNA synthesis quality control with a semi-automated dimethoxytrityl
cation assay, Biotech-niques 16, 408.
<li>	B. S. Sproat and W. Bannwarth, 1983, Improved synthesis of
oligonucleotides on controlled pore glass using phospho-triester chemistry
and a flow system, Tetrahedron Letters 24, 5771-5774.
<li>	S. P. Adams, K. S. Kavka, E. J. Wykes, S. B. Holder, and G. R. Galluppi,
1983, Hindered dialkylamino nucleoside phosphite reagents in the synthesis
of two DNA 5l-mers, J. Amer. Chem. Soc. 105, 661-663.
<li>	M. H. Caruthers, 1987, DNA synthesis for nonchemists: The
phosphoramidite method on silica supports, in Synthesis and Applications of
DNA and RNA, ed. Saran A. Narang, Academic Press, Orlando, FL.
<li>	M. H. Caruthers, 1991, Chemical synthesis of DNA and DNA analogues,
Acc. Chem. Res. 24, 278-284.
<li>	M. H. Caruthers, 1985, Gene synthesis machines: DNA chemistry and its
uses, Science 230, 281-285.
<li>	M. S. Urdea, 1987, Design, chemical synthesis, and molecular cloning of
a gene for human epidermal growth factor, Methods in Enzymology 146, 22-41.
<li>	L. Ferretti, S. S. Karnik, H. G. Khorana, M. Nassal, and D. D. Oprian,
1986, Total synthesis of a gene for bovine rhodopsin, Proc. Natl. Acad.
Sci. 83, 599-603.
<li>	M. S. Urdea, J. P. Merryweather, G. T. Mullenbach, D Coit, U.
Heberlein, P. Valenzuela, and P. J. Barr, 1983, Chemical synthesis of a
gene for human epidermal growth factor urogastrone and its expression in
yeast, Proc. Natl. Acad. Sci. 80, 7461-7465.
<li>	P. Wright, D. Lloyd, W. Rapp, and A. Andrus, 1993, Large scale
synthesis of oligonucleotides via phosphoramidite nucleosides and a high
loaded polystyrene support, Tetrahedron Letters 34, 3373-3376.
<li>	T. Tanaka and R. L. Letsinger, 1982, Syringe method for the stepwise
chemical synthesis of oligonucleotides. Nucl. Acids. Res. 10, 3249-3260.
<li>	Low-loading CPG can be obtained in the USA from Chem-Genes Corp., 925
Webster St., Needham, MA, 02192, Tel: (617) 449-5051; and in Canada from
Hukabel Scientific Ltd., C.P. 5310, Succ. St-Laurent, Ville St. Laurent,
PQ, H4L 4Z8, Tel: (514) 748-1860.
<li>	D. P. Bartel, and J. W. Szostak, 1993, Isolation of new ribozymes from
a pool of random sequences, Science 261, 1411-1418.
<li>	G. Zon, K. A. Gallo, C. J. Samson, K.-L. Shao, M. F. Summers, and R. A.
Byrd, 1985, Analytical studies of "mixed sequence"
oligodeoxyribonucleotides synthesized by competitive coupling of either
methyl- or b-cyanoethyl-N,N-diisopropylamino phosphoramidite reagents,
including 2'-deoxyinosine, Nucl. Acids. Res. 13, 8181-8196.
<li>	N. D. Sinha, 1993, Large scale oligonucleotide synthesis using the
solid-phase approach, in Methods in Molecular Biology 20: Protocols for
Oligonucleotides and Analogs, ed. S. Agrawal, Humana Press Inc., Totowa,
NJ.
<li>	R. B. Ciccarelli, P. Gunyuzlu, J. Huang, C. Scott, and F. T. Oakes,
1991, Construction of synthetic genes using PCR after automated DNA
synthesis of their entire top and bottom strands, Nucleic Acids Res. 19,
6007-6013.
<li>	D. B. McCormick and J. R. Roth, 1970, Specificity, stereochemistry, and
mechanism of the color reaction between p-dimethylaminocinnamaldehyde and
biotin analogs, Anal. Biochem. 34, 226-236.





<hr>

<p>Return to the <A HREF="http://www.abrf.org">The ABRF Home Page</A><p>
<hr>
Created: 27th July 1995<br>
Last modified: 27th July 1995<p>




</body></html>

